src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -925,14 +943,13 @@ internal virtual bool SupportsExtensions
         /// </summary>
         internal virtual void GetCandidateExtensionMethods(
             ArrayBuilder<MethodSymbol> methods,
-            string name,
+            string? name,
```

Comment:
>? [](http://example.com/codeflow?start=18&length=1)

Why is this change necessary? I didn't see any of the call sites changed/added

Reply:
This is annotating the method as currently implemented